resources/public/api/conf/6.0/loss_claims_delete.yaml

```diff
@@ -1,83 +1,84 @@
-summary: Delete a Loss Claim
-description: |
-  This endpoint allows a developer to delete an existing Loss Claim.
-  A National Insurance number and Claim ID must be provided.
+delete:
+  summary: Delete a Loss Claim
+  description: |
+    This endpoint allows a developer to delete an existing Loss Claim.
+    A National Insurance number and Claim ID must be provided.
```

Comment:
Please update this description to include tax year i.e. `A National Insurance number, tax year and Claim ID must be provided.` See the specs for reference.